samples/dashboards/demo/mqtt/demo.js

```diff
@@ -0,0 +1,1540 @@
+// - the packet timestamp is never updated
+// - real data interval does not correspond the interval given in 'hist.res
+const dataBugWorkaround = true;
+
+// Default map zoom level
+const defaultZoom = 11;
+
+// Unit for generated power (megawatts)
+const powerUnit = 'MW';
+
+// Options log (comment out/remove to disable)
+const log = console.log;
+
+// Currently active power plant/generator
+const activeItem = {
+    fullName: '',
+    plantName: '',
+    generatorId: 1,
+    connector: null
+};
+let discoveryConnector;
+
+// Configuration for Sognekraft MQTT broker
+const mqttLinkConfig = {
+    host: 'mqtt.sognekraft.no',
+    port: 8083,
+    user: 'highsoft',
+    password: 'Qs0URPjxnWlcuYBmFWNK',
+    useSSL: true
+};
+
+// Mapping of MQTT topics to data connectors
+let nDiscoveredTopics = 0;
+const topicMap = {
+    'prod/DEMO_Highsoft/kraftverk_1/overview': 'mqtt-data-1',
+    'prod/DEMO_Highsoft/kraftverk_2/overview': 'mqtt-data-2'
+};
+// Default connector
+const defaultConnId = 'mqtt-data-1';
+
+
+// Information about the received measurement data
+const measInfo = measDataInfo();
+
+// Configuration for power plant
+const powerPlantConfig = {
+    // Fields to display in tooltip and info table
+    infoFields: ['q_turb', 'q_min', 'q_max', 'P_gen', 'P_max'],
+
+    // Visual appearance on map
+    mapMarker: {
+        symbol: 'circle',
+        radius: 10,
+        fillColor: 'green'
+    }
+};
+
+// Configuration for water reservoir
+const reservoirConfig = {
+    // Fields to display in tooltip
+    tooltipFields: ['h', 'volume', 'drain', 'net_flow'],
+
+    // Fields to display in info table
+    infoFields: [
+        'volume', 'drain', 'energy',
+        'h', 'LRV', 'HRV', 'net_flow'
+    ],
+
+    // Visual appearance on map
+    mapMarker: {
+        symbol: 'mapmarker',
+        radius: 8,
+        fillColor: '#33c'
+    }
+};
+
+// Configuration for water intake
+const intakeConfig = {
+    // Fields to display in tooltip and info table
+    infoFields: ['q_min_set', 'q_min_act'],
+
+    // Visual appearance on map
+    mapMarker: {
+        symbol: 'triangle-down',
+        radius: 6,
+        fillColor: 'red'
+    }
+};
+
+// Common chart options for Highcharts components
+const commonChartOptions = {
+    title: {
+        text: ''
+    },
+    legend: {
+        enabled: false
+    }
+};
+
+
+// HTML component for displaying power plant, reservoir and
+// water intake parameters. If a description of the power plant
+// is available, it is displayed here.
+const infoComponent = {
+    type: 'HTML',
+    renderTo: 'el-info',
+    chartOptions: {
+        chart: {
+            styledMode: false
+        }
+    }
+};
+
+// Highcharts map with points for power plant, reservoirs and intakes.
+const mapComponent = {
+    type: 'Highcharts',
+    renderTo: 'el-map',
+    chartConstructor: 'mapChart',
+    title: '',
+    chartOptions: {
+        ...commonChartOptions,
+        chart: {
+            styledMode: false
```

Comment:
Tiled web maps look better when they fill the space

```suggestion
            styledMode: false,
            margin: 0
```